app/clients/sms/twilio.py

```diff
@@ -0,0 +1,84 @@
+import json
+from time import monotonic
+
+from app.clients.sms import (SmsClient, SmsClientResponseException)
+from twilio.rest import Client
+from twilio.base.exceptions import TwilioRestException
+
+class TwilioUtils:
+
+    @staticmethod
+    def get_error_from_code(error_code):
+        with open('app/clients/sms/twilio_error_codes.json') as f:
```

Comment:
this file is huge, is there a way to trim it down to just the error codes for programmable SMS?

Reply:
@matthewford I tried to find an SMS-only version of the file but don't think it exists. I agree the file is huge, though `python` consumes it (almost) instantaneously.